src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -894,6 +916,9 @@ TEST_F(ApplicationManagerImplTest,
   app_manager_impl_->OnDeviceSwitchingStart(switching_device,
                                             non_switching_device);
 
+  EXPECT_CALL(*mock_rpc_service_, ManageHMICommand(_)).WillOnce(Return(false));
```

Comment:
@ValeriiMalkov called unregistration.

Reply:
@LuxoftAKutsan Fixed in commit cb0fa9a5bd7649dc7322d420554dc10fdaf4adff